packages/test-framework/src/fixture/pageObject/Checkout.ts

```diff
@@ -92,4 +92,152 @@ export class Checkout {
         await this.page.locator('[data-test="redeemableEntry-input"]').fill(couponCode);
         await this.page.locator('[data-test="redeemableEntry-submit"]').click();
     }
+
+    async completeCustomerStepAsGuest(): Promise<void> {
+        const email = `test@example.com`;
+
+        await this.page
+            .locator('[data-test="checkout-customer-guest"]')
+            .waitFor({ state: 'visible' });
+        await this.page.locator('[data-test="checkout-customer-guest"] #email').fill(email);
+        await this.page.locator('[data-test="customer-continue-as-guest-button"]').click();
+    }
+
+    async completeCustomerStep(email: string, password: string): Promise<void> {
+        await this.page
+            .locator('[data-test="checkout-customer-guest"]')
+            .waitFor({ state: 'visible' });
+        await this.page.locator('#checkout-customer-login').click();
+        await this.page
+            .locator('[data-test="checkout-customer-returning"]')
+            .waitFor({ state: 'visible' });
+
+        await this.page.locator('[data-test="checkout-customer-returning"] #email').fill(email);
+        await this.page
+            .locator('[data-test="checkout-customer-returning"] #password')
+            .fill(password);
+        await this.page.locator('#checkout-customer-continue').click();
+    }
+
+    async fillAddressForm({
+        formId,
+        addressAppendText = '',
+    }: {
+        formId: string;
+        addressAppendText: string;
+    }): Promise<void> {
+        const address = {
```

Comment:
Can we make the address data as a prop so that devs can use their own address for testing when they try to `completeShippingAddressForm`?